### PR TITLE
(VDB-1430) Add ZRX and KNC

### PR DESCRIPTION
**Pull request by rmulhol**
- add ZRX address to config for existing transformers
- add new transformers for ZRX storage
- update OSM ABIs to match Etherscan
- update constants to include ZRX
- update integration tests where new constants cause failure

### CODE REVIEW

**Review by paytonrules — APPROVED** [review]

File: transformers/shared/constants/external.go [hunk 1]
```diff
@@ -72,10 +72,10 @@ func GetTransformerContractNames(transformerLabel string) []string {
 // GetABIFromContractsWithMatchingABI gets the ABI for multiple contracts from config
 // Makes sure the ABI matches for all, since a single transformer may run against many contracts.
 func GetABIFromContractsWithMatchingABI(contractNames []string) string {
-	initConfig()
 	if len(contractNames) < 1 {
 		logrus.Fatalf("No contracts to get ABI for")
 	}
+	initConfig()
 	contractABI := getContractABI(contractNames[0])
```
Comment (elizabethengelman) [inline transformers/shared/constants/external.go:79]: 👏

File: transformers/storage/flip/initializers/knc_flip/initializer.go [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package knc_flip
```
Comment (elizabethengelman) [inline transformers/storage/flip/initializers/knc_flip/initializer.go:1]: Do we need to also include a ZRX flip initializer?

File: transformers/storage/median/initializers/median_knc/initializer.go [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package median_knc
+
+import (
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/storage/median/initializers"
+)
+
+var StorageTransformerInitializer = initializers.GenerateStorageTransformerInitializer(constants.GetContractAddress("MEDIAN_WBTC"))
```
Comment (elizabethengelman) [inline transformers/storage/median/initializers/median_knc/initializer.go]: Should this be getting the `MEDIAN_KNC` address instead of `MEDIAN_WBTC`?

**Review by elizabethengelman — APPROVED** [review]

File: transformers/storage/median/initializers/median_knc/initializer.go (hunk 3 above)
Comment (rmulhol) [reply transformers/storage/median/initializers/median_knc/initializer.go]: yes, nice catch!

File: transformers/storage/flip/initializers/knc_flip/initializer.go (hunk 2 above)
Comment (rmulhol) [reply transformers/storage/flip/initializers/knc_flip/initializer.go:1]: we already had one due to it existing in earlier Kovan deploys 🎉

**Pull request merged**